BankingSystem/BankingSystem/Administrator.cs

```diff
@@ -0,0 +1,158 @@
+                Console.WriteLine("No transfers has been sent");
+                Console.WriteLine("!!!!!!!!!!!!!!!!!");
+            }
+        }
+
+        public void CashTransfer(string from, string to, string nameTransfer, float cash, DateTime date)
+        {
+            Account accontfrom = getAccountByName(from);
+            Account accontTo = getAccountByName(to);
+
+            accontfrom.AccountValue = accontfrom.AccountValue - cash;
+            accontTo.AccountValue = accontTo.AccountValue + cash;
+
+            informAboutTransfer(accontTo.AccountName, nameTransfer, cash, date, accontfrom.AccountName);
+
+            HistoryTransfer transferHistoryDomestic = new HistoryTransfer(date, accontfrom.AccountNumber.ToString(), accontTo.AccountNumber.ToString(),
+                                                            nameTransfer, cash, "Domestic");
+
+            _history.Add(transferHistoryDomestic);
+        }
+
+        public void CashTransferOut(string from, string to, string nameTransfer, float cash, DateTime date)
+
+        {
+            Account accontfrom = getAccountByName(from);
+
+            accontfrom.AccountValue = accontfrom.AccountValue - cash;
+
+            informAboutTransfer(to, nameTransfer, cash, date, accontfrom.AccountName);
+
+            HistoryTransfer transferHistoryOutgoing = new HistoryTransfer(date, accontfrom.AccountNumber.ToString(), to,
+                                                           nameTransfer, cash, "Outgoing");
+
+            _history.Add(transferHistoryOutgoing);
+        }
+
+        private void informAboutTransfer(string to, string nameTransfer, float cash, DateTime date, string accontfrom)
+        {
+            Console.WriteLine();
+            Console.WriteLine("\t Transfer summary:");
+            Console.WriteLine("-----------------------------");
+            Console.WriteLine($"Name account from: {accontfrom} ");
+            Console.WriteLine($"Name account to: {to}");
+            Console.WriteLine($"Name of the transfer: {nameTransfer}");
+            Console.WriteLine($"Transfer value: " + "{0:C} ", cash);
+            Console.WriteLine($"Date of the transfer: {date}");
+            Console.WriteLine("-----------------------------");
+        }
+
+        public bool CheckIfPossibleTransfer(string from, float cash)
+        {
+            Account accontfrom = null;
+            bool possible = false;
+
+            foreach (Account toSearch in _accounts)
+            {
+                if (toSearch.AccountName == from)
+                {
+                    accontfrom = toSearch;
+                    break;
+                }
+            }
+
+            if (cash <= accontfrom.AccountValue)
+            {
+                possible = true;
+
+            }
+
+            return possible;
+        }
+
+        public bool CheckIfAccountExistByName(string nametoSearch)
+        {
+            bool found = false;
```

Comment:
the method can be simplified into one-liner: `return getAccountByName(nametoSearch) != null;`